docs/developer/stock-allocation.mdx

```diff
@@ -128,6 +128,37 @@ the [`updateWarehouse`](developer/api-reference/mutations/update-warehouse.mdx)
 There is an option to change the following warehouse fields: `name`, `slug`, `email`, `address` and
 also define privacy and `click and collect` options.
 
+The warehouse could be defined as private or public. When the warehouse is public, the stock
+is shown, otherwise it isn't.
+
+### Click-and-collect warehouses
+
+The `click and collect` options allowed defining the warehouse as a collection point.
+Three options can be used:
+
+- `local stock` - the stock from this warehouse is used during the fulfillment process,
+  this option is only available for public warehouses
+- `all warehouses` - the global stock for a given product is used during the fulfillment
+  process, the available quantity is not restricted by the quantity of this warehouse,
+  and ordered products can be shipped from a different warehouse
+- `disabled` - the warehouse cannot be used as the click and collect point
+
+In the case of click-and-collect, warehouse availability does not depend on the shipping zones,
+it's only restricted by channel relation.
+So the warehouse is available for the given order as a collection point when the following two
+conditions are fulfilled:
+
+- there is a stock with available quantity for a given product
+- the warehouse is available in the channel in which the order is created
```

Comment:
In the case of `click-and-collect`, the warehouse availability does not depend on the shipping zones. It is limited only by the given channel.
The warehouse is available as a collection point for the given order under two conditions:
- there is a stock with available quantity for a given product
- the warehouse is available in the channel in which the order is created.